src/app/settings/organizations.component.html

```diff
@@ -74,6 +74,16 @@ <h1>
                                 <i class="fa fa-cog fa-lg" aria-hidden="true"></i>
                             </button>
                             <div class="dropdown-menu dropdown-menu-right">
+                                <a *ngIf="!o.isResetPasswordEnrolled" class="dropdown-item" href="#" appStopClick
```

Comment:
should we show some kind of icon on the org list indicating this status?

Reply:
Added! Emulated the current Send status icons (for now)